_config.yml

```diff
@@ -1 +0,0 @@
-include: [".well-known"]

```

Comment:
We need to add this to the website repo.

Reply:
Happy to, just need to know where in the structure?

Reply:
Isnt it already there? https://github.com/zaproxy/zaproxy-website/blob/master/site/static/.well-known/security.txt

Reply:
Ya, I was assuming he was referring to the Jekyll config file. So I PRd that.

Reply:
Yes, it was the config file.